.github/workflows/ci.yaml

```diff
@@ -25,6 +25,7 @@ jobs:
           "e2e/smoke_nickel_1_1_1",
           "e2e/smoke_nickel_1_4_0",
           "examples/hello_world"
```

Comment:
```suggestion
          "examples/hello_world",
```

Reply:
This breaks CI, but unfortunately not loudly.

Reply:
Good catch, thanks for finding it. Let me fix the CI now.